### DISCUSSION TITLE
help me about YOLOX-POSE

### DISCUSSION THREAD

**Discussion by ghost** [open-mmlab/mmpose#2226] [Q&A] [answered]
Hi there.

I tried to use YOLOX-POSE along this [USAGE](https://github.com/open-mmlab/mmpose/tree/main/projects/yolox-pose)

Could someone please teach me how to solve this problem????

![Screenshot from 2023-04-14 08-53-31](https://user-images.githubusercontent.com/126233717/231909878-f87df639-cc33-48b5-8f5e-1d6baca07ce8.png)

**Comment by Ben-Louis** [comment, marked as answer]
Hi, thanks for using MMPose. Have you set the `PYTHONPATH` as illustrated in the README?
![image](https://user-images.githubusercontent.com/26127467/231960686-26eecbe9-14b9-4cb6-84f2-954019a200e3.png)

**Reply by ghost** [reply]
Hi, thank you for reply.  I'm done!!
![Screenshot from 2023-04-15 15-08-42](https://user-images.githubusercontent.com/126233717/232188957-5a91e4fa-beb4-48a2-8b0a-448fb9103262.png)
